subprojects/test-runner/test-robolectric-inhouse-runner/build.gradle.kts

```diff
@@ -0,0 +1,11 @@
+plugins {
+    id("convention.kotlin-jvm")
+    id("convention.publish-kotlin-library")
+}
+
+dependencies {
+    api(projects.subprojects.testRunner.testRobolectricRunner)
+
+    implementation(projects.subprojects.logger.logger)
+    implementation(projects.subprojects.testRunner.shared.loggerProviders)
+}
```

Comment:
Why do we need two modules?
- test-runner/test-robolectric-inhouse-runner
- test-robolectric-runner

I didn't get the intention by the code or by dependencies.
What's the difference in terms of responsibility and knowledge?

Reply:
Definitely not at this stage. I separated it into several modules in the likeness to the instrumentation runner. 
You are right, I'll merge it for now and divide later if necessary